angular/src/app/workspace/simple-search/simple-search.component.html

```diff
@@ -44,14 +44,14 @@
             </button>
           </mat-menu>
         </div>
-        <div>
-          <button type="submit"
-                  mat-flat-button
-                  color="primary">
-            Filter
-          </button>
-        </div>
       </form>
+      <div class="submit-container">
```

Comment:
**Change request:** The look and feel for this part of the UI is usually done with `<mat-toolbar>` with class `tertiary-toolbar`.

```html
<mat-toolbar class="tertiary-toolbar">
      <mat-toolbar-row>...<mat-toolbar-row>
</mat-toolbar>
```

You may need to override the CSS to make sure that the button is on the right. In that case, `flex-direction: row-reverse` might help you.